frontend/src/metabase/entities/bookmarks.js

```diff
@@ -1,7 +1,7 @@
 import { createEntity } from "metabase/lib/entities";
 import Collection from "metabase/entities/collections";
 import Dashboard from "metabase/entities/dashboards";
-import Question from "metabase/entities/questions";
+import Questions from "metabase/entities/questions";
```

Comment:
nip: if you're renaming it to `Questions` (plural), should `Collection` and `Dashboard` be plural too?

Reply:
Yup, will apply one standard to these in follow up PRs.